sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessSessionEventArgs.cs

```diff
@@ -34,6 +34,16 @@ public class ProcessSessionEventArgs : EventArgs
         /// </summary>
         public DateTimeOffset SessionLockedUntil => _sessionReceiver.SessionLockedUntil;
 
+        /// <summary>
+        /// The entity path corresponding to the entity that the session is defined on.
+        /// </summary>
+        public string EntityPath => _sessionReceiver.EntityPath;
+
+        /// <summary>
+        /// The fully qualified namespace of the Service Bus namespace that the session is defined on.
```

Comment:
```suggestion
        /// The fully qualified Service Bus namespace that the message was received from.
```

This read a bit awkwardly to me; I used the [Receiver comment](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs#L28) as the basis for the rewording.